### ISSUE TITLE
Create a man page

### ISSUE THREAD

**Issue by Airblader** [fmthoma/vgrep#38]
The title is pretty self-explanatory; tools like these definitely should have a man page. :-)

**Comment by fmthoma** [comment]
@Airblader I agree. Since I've never created a `man` page yet, and since I'm not familiar with the format and the tooling around it, it might take a while for me. Any help would be appreciated!

**Comment by Airblader** [comment]
The easy answer is to write the raw man file yourself. An example for this is [i3lock's man page](https://github.com/i3/i3lock/blob/master/i3lock.1). But that's a bit bothersome. Another way is [this](https://github.com/i3/i3status/tree/master/man), using asciidoctor and then generating the man page from it. But I'm not familiar with Haskell build system setup.

I assume other Haskell projects have man pages and I'd take inspiration from there. :-)

**Comment by sjakobi** [comment]
pandoc can generate manpages, but I have no experience with that. [Here](http://jbl.web.cern.ch/jbl/doc/manpages/)'s a short tutorial.
